test/e2e/basics/popup.test.js

```diff
@@ -21,13 +21,14 @@ test.describe("Popup test", () => {
     });
 
     test("Turn on options", async ()=>{
-        for(let i = 0; i < 3; i++){
-            await page.keyboard.press("Tab");
-        }
-        for(let i = 0; i < 2; i++){
-            await page.keyboard.press("Tab");
-            await page.keyboard.press("Space");
-        }
+
+        await page.keyboard.press("Tab");
+        await page.keyboard.press("Tab");
+        await page.keyboard.press("Tab");
+        await page.keyboard.press("Tab");
+        await page.keyboard.press("Tab");
+        await page.keyboard.press("Space");
```

Comment:
You don't have to action this here, but just a comment:  I find the loops over tabs in tests really hard to read.  I prefer the linear code that you've put here, one tab after another, then a space.  I think that what could improve it further would be to add a comment about which thing is focused after each tab.  This helps someone who comes along after read and understand the test quickly. i.e. yourself, six months from now.